src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/client/PrisonApiClient.kt

```diff
@@ -26,11 +26,11 @@ class PrisonApiClient(
       .block(apiTimeout)
   }
 
-  fun getPrisonerDetails(offenderNo: String): PrisonerDetailDto? {
+  fun getPrisonerHousingLocation(offenderNo: String): PrisonerHousingLocationsDto? {
     return webClient.get()
-      .uri("/api/prisoners/$offenderNo/full-status")
+      .uri("/api/offenders/$offenderNo/housing-location")
       .retrieve()
-      .bodyToMono(prisonerDetail)
+      .bodyToMono(prisonerHousingLocationsDto)
```

Comment:
The use of bodyToMono can cause problems I think, if nothing comes back from the API you will get an empty object,... is that what we need. But since we are already using this in other calls maybe we will wait to see if we get any problems and fix as and when.

Reply:
Yes, this would be something we will need to look at at a later stage.